src/test/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTripTest.java

```diff
@@ -884,6 +884,65 @@ void testDuplicateNotifications() throws Exception {
         Persistence.otpUsers.removeById(observer.id);
     }
 
+    @Test
+    void testCheckMonitoredTripWhenUTCIsNextDay() throws Exception {
+        MonitoredTrip monitoredTrip = PersistenceTestUtils.createMonitoredTrip(
+            user.id,
+            OtpTestUtils.OTP2_DISPATCHER_PLAN_RESPONSE.clone(),
+            false,
+            OtpTestUtils.createDefaultJourneyState()
+        );
+
+        OtpTestUtils.updateBaseItineraryTime(
+            monitoredTrip.itinerary,
+            DateTimeUtils.makeOtpZonedDateTime(monitoredTrip.itinerary.startTime)
+                .withHour(19)
+                .withMinute(15)
+        );
+        monitoredTrip.itineraryExistence.monday = new ItineraryExistence.ItineraryExistenceResult();
+        monitoredTrip.itineraryExistence.tuesday = new ItineraryExistence.ItineraryExistenceResult();
+        monitoredTrip.tripTime = "19:15";
+        monitoredTrip.leadTimeInMinutes = 30;
+        Persistence.monitoredTrips.create(monitoredTrip);
+        LOG.info("Created trip {}", monitoredTrip.id);
+
+        // Set up an OTP mock response in order to trigger some of the monitor checks.
+        OtpResponse mockResponse = mockOtpPlanResponse();
+        Itinerary mockItinerary = mockResponse.plan.itineraries.get(0);
+
+        // change time to be greater than 30 min lead time
+        DateTimeUtils.useFixedClockAt(
+            noonMonday8June2020
+                .withDayOfMonth(9)
+                .withHour(17)
+                .withMinute(50)
+        );
+
+        // Next, run a monitor trip check from the new monitored trip using the simulated response.
+        CheckMonitoredTrip checkMonitoredTrip = new CheckMonitoredTrip(monitoredTrip, () -> mockResponse);
```

Comment:
Replace `() -> mockResponse` with `this::mockOtpPlanResponse` (2 instances in this method)